modules/logging_configs/secrets.tf

```diff
@@ -0,0 +1,19 @@
+resource "aws_ssm_parameter" "datadog_key" {
+  name   = "/${var.ssm_prefix}/${var.environment_name}/DATADOG_KEY"
+  type   = "SecureString"
+  value  = "ChangeMeInTheWebConsole"
+  key_id = aws_kms_key.for_ssm_params.key_id
```

Comment:
I think we've moved this outta here right?

Reply:
YES good catch

Reply:
Removed the file as we have no secrets being generated from this module